problem.py

```diff
@@ -19,7 +19,7 @@ class Problem:
         fib_quotient is ... of quotient  (111)
     """
 
-    def __init__(self, lower=4, upper=10):
+    def __init__(self, lower=10, upper=15):
```

Comment:
did you want to commit this change?

Reply:
No, but we can undo this later, probably want to use variables in SETTINGS for this as well. We can later use a structure .yml file for these setting variables so there's an organized hierarchy